platform-includes/sourcemaps/upload/primer/javascript.sveltekit.mdx

```diff
@@ -0,0 +1,82 @@
+### Source Maps Upload
+
+By default, `sentrySvelteKit()` will add an instance of the Sentry Vite Plugin, to upload source maps for both server and client builds. This means that when you run a production build (`npm run build`), source maps will be generated and uploaded to Sentry so that you get readable stack traces in your Sentry issues.
+
+However, you still need to specify your Sentry auth token as well as your org and project slugs.
+There are two ways to set them:
+
+**Option 1**
```

Comment:
m: The options here really only differ in how much users set as env variables vs. directly in code. Users can set project and org slugs in code but they _should never_ set the auth token in code as it must remain a secret and shouldn't be committed. 

So I'd recommend we:
- always show the `.env` file for setting the `SENTRY_AUTH_TOKEN`
- also show a complete `.env` file for option 1 (see my comment)
- show both `.env` and `vite.config.ts` for option 2 (basically how we used to show it in manual setup).

Does this make sense?